apps/networkmonitor/networkmonitor.nim

```diff
@@ -429,7 +441,8 @@ proc initAndStartApp(
 
   nodeBuilder.withNodeKey(key)
   nodeBuilder.withRecord(record)
-  nodeBuilder.withPeerManagerConfig(maxRelayPeers = none(int), shardAware = true)
+  nodeBUilder.withSwitchConfiguration(maxConnections = some(MaxConnectedPeers))
+  nodeBuilder.withPeerManagerConfig(maxRelayPeers = some(20), shardAware = true)
```

Comment:
The shard aware peer manager will try to keep each shard populated with the same average # of peers.
It prevent too many in connections and also keep a % of max connection for service peers.
Are you sure that's what you want in this case?

Reply:
Good question, I am not sure - what do you think?:) I probably want same average # of peers per shard. I don't really need any in connections since the main goal of NM is to discover nodes and keep track of the network, but not really to be discoverable by others, so limiting in connections might be useful? 

I'll probably metge this PR and we can think about it for next one